flang/lib/Lower/OpenMP.cpp

```diff
@@ -22,6 +22,7 @@
 #include "flang/Semantics/tools.h"
 #include "mlir/Dialect/OpenMP/OpenMPDialect.h"
 #include "llvm/Frontend/OpenMP/OMPConstants.h"
+#include <iostream>
```

Comment:
IIRC, iostream is a no-no in LLVM.

Reply:
The good news is that this line looks unnecessary.

Reply:
Oops, remnants of debugging. Update coming in a bit!